contracts/VaultV3.vy

```diff
@@ -200,15 +200,14 @@ enum Rounding:
     ROUND_DOWN
     ROUND_UP
 
-# IMMUTABLE #
+# STORAGE #
 # Underlying token used by the vault.
-ASSET: immutable(ERC20)
+asset: public(address)
 # Based off the `asset` decimals.
-DECIMALS: immutable(uint256)
+decimals: public(uint256)
 # Deployer contract used to retrieve the protocol fee config.
-FACTORY: public(immutable(address))
+FACTORY: public(address)
```

Comment:
should this be lower cased as well?

Reply:
Technically yes, but the strategy's have it as upper case and same with the previous versions.

So kept it upper case to keep all the ABI's constant

ASSET and DECIMALS weren't public so it didnt matter for the ABI

Reply:
I guess the storage variable could be lower case and internal, then make a getter that matches for the ABI?

Reply:
Changed it to be lower case and add a separate getter https://github.com/yearn/yearn-vaults-v3/pull/191/commits/505e4df9e7be252ad831b732b491cb60729e5cce